pkg/limits/ingest_limits.go

```diff
@@ -32,24 +32,16 @@ const (
 )
 
 var (
-	tenantPartitionDesc = prometheus.NewDesc(
+	partitionsDesc = prometheus.NewDesc(
 		constants.Loki+"_ingest_limits_partitions",
-		"The current number of partitions per tenant.",
```

Comment:
I don't believe this is a meaningful metric, it just adds unnecessary cardinality. The same pod cannot consumer different partitions for different tenants.